pkg/kubelet/cri/streaming/portforward/httpstream.go

```diff
@@ -202,26 +202,26 @@ func (h *httpStreamHandler) requestID(stream httpstream.Stream) string {
 			requestID = strconv.Itoa(int(stream.Identifier()) - 2)
 		}
 
-		klog.V(5).InfoS("Connection automatically assigning request ID from stream type and stream ID", "connection", h.conn, "request", requestID, "streamType", streamType, "stream", stream.Identifier())
+		klog.V(5).InfoS("Connection automatically assigning request ID from stream type and stream ID", "request", requestID, "streamType", streamType, "stream", stream.Identifier())
 	}
 	return requestID
 }
 
 // run is the main loop for the httpStreamHandler. It processes new
 // streams, invoking portForward for each complete stream pair. The loop exits
 // when the httpstream.Connection is closed.
-func (h *httpStreamHandler) run() {
-	klog.V(5).InfoS("Connection waiting for port forward streams", "connection", h.conn)
+func (h *httpStreamHandler) run(ctx context.Context) {
+	klog.V(5).InfoS("Connection waiting for port forward streams")
 Loop:
 	for {
 		select {
 		case <-h.conn.CloseChan():
```

Comment:
should there be a `case <-ctx.Done():` added that breaks out and returns?

Reply:
related to https://github.com/kubernetes/kubernetes/pull/114854/commits/d856ef689ba8d4010f41da18bc02f1191c854f3f#r1063541868